cmd/cmd_utils.go

```
@@ -605,3 +633,42 @@ func getConfigAndStacksInfo(commandName string, cmd *cobra.Command, args []strin
+	}
+	showUsageExample(cmd, details)
+}
+
+func showUsageExample(cmd *cobra.Command, details string) {
+	contentName := strings.ReplaceAll(cmd.CommandPath(), " ", "_")
+	suggestion := fmt.Sprintf("\n\nRun `%s --help` for usage", cmd.CommandPath())
+	if exampleContent, ok := examples[contentName]; ok {
+		suggestion = exampleContent.Suggestion
+		details += "\n## Usage Examples:\n" + exampleContent.Content
+	}
+	u.PrintErrorMarkdownAndExit("Incorrect Usage", fmt.Errorf(details), suggestion)
```

golangci-lint (1.62.2)

printf: non-constant format string in call to fmt.Errorf

(govet)